src/nxt_main_process.c

```diff
@@ -1245,11 +1271,11 @@ nxt_main_port_modules_handler(nxt_task_t *task, nxt_port_recv_msg_t *msg)
 
     rt = task->thread->runtime;
 
-    if (msg->port_msg.pid != rt->port_by_type[NXT_PROCESS_DISCOVERY]->pid) {
+    if (msg->pid != rt->port_by_type[NXT_PROCESS_DISCOVERY]->pid) {
```

Comment:
Although it is not related to your changes, would be nice to add alert message here.

Reply:
done